google/cloud/forseti/services/inventory/base/gcp.py

```diff
@@ -41,7 +41,7 @@ class ApiClient(object):
     __metaclass__ = abc.ABCMeta
 
     @abc.abstractmethod
-    def fetch_bigquery_dataset_policy(self, project_number, dataset_id):
+    def fetch_bigquery_dataset_policy(self, project_id, project_number, dataset_id):
         """Dataset policy Iterator for a dataset from gcp API call.
 
         Args:
```

Comment:
Please add 'project_id (str): id of the project to query.' to the args section, here and below

Reply:
Updated.